src/composables/pools/useTokenApprovals.ts

```diff
@@ -20,7 +20,7 @@ export type ApprovalStateMap = {
 
 export default function useTokenApprovals(
   tokenAddresses: string[],
-  amounts: Ref<string[]>
+  amounts: string[]
```

Comment:
Just making sure this definitely won't break things? If the amounts change after this composable is initiated will that affect the state? Or is this composable only initialised after the amounts have been set?

Reply:
My assumption when i was coding this was the ref isn't needed because it's only accessed by amounts.value inside this composable. However it is inside a ref and computed value so it may actually be reactive and change. Will change this back. 

It doesn't matter in this instance because the amounts passed in are fixed, but it may mater in other instances in the future if the user can change the amount after this composable has been created.